packages/lib/ren/src/config.ts

```diff
@@ -37,4 +37,9 @@ export interface RenJSConfig {
      * Allow overriding the gPubKey.
      */
     gPubKey?: Buffer;
+
+    /**
+     * Ensure the V2 RPC is used for all assets
+     */
+    forceV2RPC?: Buffer;
```

Comment:
What do you think of using something like `useV2TransactionFormat` so that devs know that it will affect things like `queryTx`. (and `use` instead of `force` just as a personal preference - although I also think it would be less confusing to come across in the docs compared to `force`).

Reply:
Good point